### DISCUSSION TITLE
Switching language is buggy

### DISCUSSION THREAD

**Discussion by 7OH** [valinet/ExplorerPatcher#606] [General]
When switching language (with ctrl-shift) it restart explorer.
Win+Space didnot show at all.

**Comment by 7OH** [comment]
Bug only if used win11 method. If using win10 method - works fine.

**Comment by valinet** [comment]
#56

**Comment by valinet** [comment]
I don't know, for me it works just fine. It can be buggy, idk, the Windows 10 UI is not really supported on Windows 11, that's the simple truth. Maybe there is some workaround, but I don't know about it. Some user said something similar a while ago. As long as you can use the legacy Windows 10 switcher just fine, I guess you could use that and that's be it, idk...

You can use these instructions: https://docs.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps to collect a full crash dump for `explorer` (when it is restarted) and maybe me/someone around here can look on that and figure out what might be happening and even come up with a fix.

Also, idk if `Ctrl`+`Shift` is really supported anymore. Try using `Win`+`Space` if you want to use the newer flyouts, I guess...

**Reply by 7OH** [reply]
Yup. Win+Space with win10 works as well as Ctrl-Shift.
With win11 - it crashed in any variant. 
So now it really helps to switch to win10 switcher.
Dow I need place log here in more datailed view ?
here some info from windows logs
"Имя сбойного приложения: explorer.exe, версия: 10.0.22523.1000, метка времени: 0xd04e4f8e
Имя сбойного модуля: TextInputExtensions.dll, версия: 521.33300.0.0, метка времени: 0x61a45427
Код исключения: 0xc0000005
Смещение ошибки: 0x000000000000561b
Идентификатор сбойного процесса: 0xe3c
Время запуска сбойного приложения: 0x01d7fc3462ca7891
Путь сбойного приложения: C:\WINDOWS\explorer.exe
Путь сбойного модуля: C:\Windows\SystemApps\MicrosoftWindows.Client.CBS_cw5n1h2txyewy\TextInputExtensions.dll
Идентификатор отчета: d0970bbe-75c8-4712-bd23-db2d0cd11a74
Полное имя сбойного пакета: 
Код приложения, связанного со сбойным пакетом: "

**Reply by 7OH** [reply]
I test it in RDP on a hyper-v - maybe it cause problem.

**Reply by valinet** [reply]
> I test it in RDP on a hyper-v - maybe it cause problem.

Yeah, maybe/probably. It has to be investigated.

Logfiles are generally useless. Crash dumps might be useful.

**Comment by 7OH** [comment]
Here dumps:
1 - with click on label ENG
2 - with ctrl+shift
3 - with win+space
[explorer_mini.zip](https://github.com/valinet/ExplorerPatcher/files/7797721/explorer_mini.zip)
(can upload full dump if needed)
